script.js

```diff
@@ -231,4 +231,27 @@ form.addEventListener('submit', (event) => {
   event.preventDefault();
   checkLowerCase(email);
 });
+
+
 // START STORAGE
+
+const valueInput = JSON.parse(localStorage.getItem('valueInput')) || [];
+
+function addVale(n, e, t) {
+  valueInput.push({ n, e, t });
+  localStorage.setItem('valueInput', JSON.stringify(valueInput));
+  return { n, e, t };
+}
+
+function showNow({ n, e, t }) {
+  namee.value = n;
+  email.value = e;
+  text.value = t;
+}
```

Comment:
- [ ] **Good job on saving the data on local storage 👍 👍** However, for the best practice, I think it would be better to give the meaningful names of your variables 😎 
   _Remember, Using meaningful names for variables can make your code easier to read and understand. When you use descriptive names, it becomes easier for other people (or even yourself, if you come back to the code later) to understand what the code is doing. It also helps to prevent errors and can make debugging easier. 👍👍_

    - Kindly use meaningful names to replace ```n, e, t``` 🙏